packages/frontend/src/components/UserSettings/AllowedDomainsPanel/index.tsx

```diff
@@ -14,6 +14,16 @@ import { useProjects } from '../../../hooks/useProjects';
 import { isValidEmailDomain } from '../../../utils/fieldValidators';
 
 const roleOptions = [
+    {
```

Comment:
so the api accepts any role as the default role? Do we want to change that?

Reply:
Added validation in the API